.github/workflows/e2e-tests-ondemand.yaml

```diff
@@ -1,20 +1,44 @@
 name: E2E tests ondemand
 
 on:
+  schedule:
+    # every Mon/Wed/Fri at 08:00 UTC
+    - cron: 0 8 * * 1,3,5
   workflow_dispatch:
     inputs:
       target:
         description: 'Target branch to run E2E tests over'
         required: true
-        # TODO: @andriisoldatenko revisit it
         default: 'release-1.1'
 
 permissions:
   checks: write
 
+# based on:
+# https://docs.dynatrace.com/docs/setup-and-configuration/technology-support/support-model-for-kubernetes
+# https://kubernetes.io/releases/
 jobs:
-  run-in-k8s-1-21:
-    name: Run in Kubernetes 1.21
+  run-matrix:
+    name: Run in Kubernetes/OCP
+    strategy:
+      matrix:
+        flc_env:
+          - dto-k8s-1-24
+          - dto-k8s-1-25
+          - dto-k8s-1-26
+          - dto-k8s-1-27
+          - dto-k8s-1-28
+          - dto-k8s-1-29
+          - dto-ocp-4-10
+          - dto-ocp-4-11
+          - dto-ocp-4-12
+          - dto-ocp-4-13
+        flc_environment: [dto-k8s-ondemand, dto-ocp-ondemand]
```

Comment:
we can just keep it split, as this will duplicate `flc_env` for both `flc_environment` entries. The only way this may have worked would be [includes](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations) or setting the `flc_environment` depending on if `flc_env` contains `ocp` or `k8s`

Reply:
I;ve tried this and it's broken 
```
      matrix:
        flc_env:
          - dto-k8s-1-24
          - dto-k8s-1-25
          - dto-k8s-1-26
          - dto-k8s-1-27
          - dto-k8s-1-28
          - dto-k8s-1-29
          - dto-ocp-4-10
          - dto-ocp-4-11
          - dto-ocp-4-12
          - dto-ocp-4-13
        flc_environment: [dto-k8s-ondemand, dto-ocp-ondemand]
        exclude:
          - flc_env: [dto-ocp-4-10, dto-ocp-4-11, dto-ocp-4-12, dto-ocp-4-13]
            flc_environment: dto-k8s-ondemand
          - flc_env: [dto-k8s-1-24, dto-k8s-1-25, dto-k8s-1-26, dto-k8s-1-27, dto-k8s-1-28, dto-k8s-1-29]
            flc_environment: dto-ocp-ondemand
```